### DISCUSSION TITLE
Unable to use beta feature "Environments"

### DISCUSSION THREAD

**Discussion by oppknox** [community/community#26617] [Actions]
<p>I come from the ADO world, trying to learn GitHub actions. Luckily I found this really great blog by Tim Heuer to get up to speed quickly. Unfortunately none of my accounts have have the Environments option. I of course can see secrets and many of the other options. I understand this feature is in beta, but is it limited to select paid offerings or do I need to opt in somewhere?  Thanks!</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/34199047/181098700-2d777885-a543-4648-9ed4-e73d71603379.png" class="site-icon" width="16" height="16">
      <a href="https://timheuer.com/blog/add-approval-workflow-to-github-actions/" target="_blank" rel="noopener nofollow ugc">timheuer.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:672/448;"><img src="https://user-images.githubusercontent.com/34199047/181098702-b24139d3-1ddd-44e3-a007-6c298d992b6c.png" class="thumbnail" width="672" height="448"></div>

<h3><a href="https://timheuer.com/blog/add-approval-workflow-to-github-actions/" target="_blank" rel="noopener nofollow ugc">Adding approval workflow to your GitHub Action</a></h3>

<p>Want to add an approval workflow to your GitHub Action deployment easily?  With new capabilities now you can!</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/actions/reference/environments#creating-an-environment" target="_blank" rel="noopener nofollow ugc">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/actions/reference/environments#creating-an-environment" target="_blank" rel="noopener nofollow ugc">Environments - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/actions/reference/environments#creating-an-environment" target="_blank" rel="noopener nofollow ugc">//docs.github.com/en/actions/reference/environments</a></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by Simran-B** [comment, marked as answer]
<p>Don’t you see the entry Environment under setting of a public repo that you own? It should be there without opting in:</p>
<blockquote>
<p>In this open beta there is no need to sign up, <strong>all existing GitHub organizations and accounts</strong> can use the new capabilities in their <strong>public repositories</strong> and <strong>GitHub Enterprise Cloud organizations</strong> can use them in <strong>all repositories</strong>.</p>
</blockquote>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.blog/wp-content/uploads/2019/01/cropped-github-favicon-512.png?fit=32%2C32" class="site-icon" width="32" height="32">
      <a href="https://github.blog/changelog/2020-12-15-github-actions-environments-environment-protection-rules-and-environment-secrets-beta/" target="_blank" rel="noopener nofollow ugc">The GitHub Blog</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/443;"><img src="https://user-images.githubusercontent.com/7819991/181098704-85cb6272-c76b-422f-bf1f-6b6c14c94996.png" class="thumbnail" width="690" height="443" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/d/9d934875b8b01850d442f5f19e34d9a56905bb06_2_690x443.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/d/9d934875b8b01850d442f5f19e34d9a56905bb06_2_1035x664.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/d/9d934875b8b01850d442f5f19e34d9a56905bb06.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/d/9d934875b8b01850d442f5f19e34d9a56905bb06_2_10x10.png"></div>

<h3><a href="https://github.blog/changelog/2020-12-15-github-actions-environments-environment-protection-rules-and-environment-secrets-beta/" target="_blank" rel="noopener nofollow ugc">GitHub Actions: Environments, environment protection rules and environment...</a></h3>

<p>GitHub Actions: Environments, environment protection rules and environment secrets (beta)</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by oppknox** [comment]
<p>Hey Simran, I just created a new public repository and I can now see Environments. Thank you!</p>
<p>This is extremely disappointing that its only available in Public and Enterprise. I would have hoped to see  see feature parity between ADO and GitHub. $21 a month per user is an insane request when its free on ADO and many other providers with many more mature features.</p>
